types/ember-data/index.d.ts

```diff
@@ -1249,15 +1249,6 @@ export namespace DS {
          * within a single runloop.
          */
         coalesceFindRequests: boolean;
```

Comment:
What about this one? It's a [public property](https://api.emberjs.com/ember-data/3.21/classes/JSONAPIAdapter/properties/coalesceFindRequests?anchor=coalesceFindRequests) that could be defined with a getter, so might as well be complete. I notice it's repeated in the `Adapter` base class, so you might have to do that one too.

Reply:
Sure, I moved it to a new `Adapter` interface. Looks ok, as al the other adapters implement it 👍